### DISCUSSION TITLE
Could you give an example about how to use Conditional Random Field (CRF) Block for image segmentation task？

### DISCUSSION THREAD

**Discussion by Fivethousand5k** [Project-MONAI/MONAI#5475] [Q&A]
Hi! Could you give an example on the usage of monai.networks.blocks.CRF for segmentation task?

**Comment by wyli** [comment]
there's a work in progress here https://github.com/Project-MONAI/tutorials/blob/f4e4c25655594fb977b71cec53af995eeb97c560/modules/conditional_random_field.ipynb https://github.com/Project-MONAI/tutorials/pull/170
